identity/webapp/pages/api/registration.ts

```diff
@@ -38,16 +36,14 @@ export default (req: NextApiRequest, res: NextApiResponse): void => {
           .then(() => {
             res.redirect(302, `/account`);
           })
-          .catch(() => {
-            res.status(400).json({
-              error: 'Registration failed',
-            });
+          .catch(error => {
+            console.error(error);
+            res.redirect(302, `/account/error`);
           });
       }
     } catch (error) {
-      res.status(400).json({
-        error: error.message,
-      });
+      console.error(error);
+      res.redirect(302, `/account/error`);
```

Comment:
Do we need both of these catch blocks?

Reply:
Think so – one for if there's an error with Axios, and the other for if there's an error decoding the JWT?